packages/records/lib/utils/encryption.ts

```diff
@@ -14,7 +14,7 @@ function isEncrypted(data: UnencryptedRecordData | EncryptedRecordData): data is
     return 'encryptedValue' in data;
 }
 
-export function decryptRecord(record: FormattedRecord): UnencryptedRecordData {
+export function decryptRecordData(record: FormattedRecord): UnencryptedRecordData {
```

Comment:
renaming to make it clearer that the function only return the unencrypted data not the entire record